pkg/apis/stork/v1alpha1/applicationbackup.go

```diff
@@ -49,8 +49,9 @@ type ApplicationBackupSpec struct {
 	BackupObjectType string `json:"backupObjectType"`
 	// SkipAutoExecRules is false by default, if set true will skip auto exec rules for VM specific backup.
 	// This field is unused for non VM specific Backup.
-	SkipAutoExecRules bool `json:"skipAutoExecRules"`
-	DirectKDMP        bool `json:"directKDMP"`
+	SkipAutoExecRules    bool     `json:"skipAutoExecRules"`
+	DirectKDMP           bool     `json:"directKDMP"`
+	ExcludeResourceTypes []string `json:"excludeResourceTypes"`
```

Comment:
@siva-portworx since your are changed the spec. Can you run `make codegen` to updated the zz_generated_file

Reply:
Sorry, I missed while porting to new branch. Fixed it.